### PR TITLE
add Nekopoi presence

### PR DESCRIPTION
**Pull request by iseizuu**

### CODE REVIEW

**Comment by iseizuu** [comment]
proof:
image

**Review by Bas950 — APPROVED** [review]

File: websites/N/Nekopoi/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    },
+    "service": "NekoPoi",
+    "description": {
+      "en": "Nekopoi is indonesian hentai and jav streaming site",
+      "id": "Nekopoi adalah tempat mencari hentai dan jav bersubtitle indonesia"
+    },
+    "url": "nekopoi.care",
+    "version": "1.0.0",
+    "logo": "https://i.imgur.com/u08RGtt.png",
+    "thumbnail": "https://i.imgur.com/GrEgIWx.png",
```
Comment (ririxidev) [inline websites/N/Nekopoi/dist/metadata.json:15]: different thumbnail with hidden nsfw things thanks :)

**Review by ririxidev — CHANGES_REQUESTED** [review]

File: websites/N/Nekopoi/dist/metadata.json (hunk 1 above)
Comment (iseizuu) [reply websites/N/Nekopoi/dist/metadata.json:15]: omg ,,, im so sorry i will change this one

Comment (Bas950) [reply websites/N/Nekopoi/dist/metadata.json:15]: lmao how did I not notice that xDDD

**Review by ririxidev — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**